CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 0.52.3      2023-11-30
+
+* Reduce frequency of collector log generation
+  - Previously, real-time snapshots would generate log lines every 10 seconds,
+    which made errors hard to find
+  - Now, a single log line is printed once a minute with a summary of snapshots
+    submitted
+  - Note that `--verbose` will still log every snapshot as it's submitted
```

Comment:
I realize that I made a related change in #482 that's probably worth mentioning, in case anyone has regular expressions on the logs set up:

```
* Collector log output: Add "full" prefix for full snapshots sent every 10 minutes
  - This changes the "Submitted snapshot successfully" message to read "Submitted full snapshot successfully" instead
```